rlberry/rendering/tests/test_rendering_interface.py

```diff
@@ -83,16 +82,36 @@ def test_render2d_interface_wrapped(ModelClass):
         env.enable_rendering()
         if env.is_online():
             for _ in range(2):
-                state = env.reset()
+                observation, info = env.reset()
                 for _ in range(5):
-                    assert env.observation_space.contains(state)
+                    assert env.observation_space.contains(observation)
                     action = env.action_space.sample()
-                    next_s, _, _, _ = env.step(action)
-                    state = next_s
+                    observation, _, _, _, _ = env.step(action)
                 env.render(loop=False)
             env.save_video("test_video.mp4")
             env.clear_render_buffer()
         try:
             os.remove("test_video.mp4")
         except Exception:
             pass
+
+
+def test_render_appelGold():
+    env = AppleGold()
+    env.render_mode = "human"
+    env = Wrapper(env)
+
+    if env.is_online():
+        for _ in range(2):
+            observation, info = env.reset()
+            for _ in range(5):
+                assert env.observation_space.contains(observation)
+                action = env.action_space.sample()
+                observation, _, _, _, _ = env.step(action)
+            env.render(loop=False)
+        env.save_video("test_video.mp4")
```

Comment:
in a temp folder ? test_video.mp4 is not an uncommon name and this may accidentally overwrite the video of some (very) unlucky user.